assets/src/scripts/web-services/graphql.js

```diff
@@ -10,20 +10,23 @@ query($bBox: String,
       $siteType: [String],
       $startDateLo: String,
       $startDateHi: String) {
-  features(bBox: $bBox,
+  allFeatures(bBox: $bBox,
```

Comment:
This corresponds to changes to the API wrapper made in 10e7743. `allFeatures` gives back the features as well as the total number of sites returned.